ddtrace/contrib/kombu/utils.py

```diff
@@ -18,7 +18,7 @@ def extract_conn_tags(connection):
         host, port = connection.host.split(":")
         return {
             net.TARGET_HOST: host,
-            net.TARGET_PORT: port,
+            "out.host": port,  ## Fix later
```

Comment:
I always worry about changing tag names because it's possible that a user is generating a metric on it or using it in an analytics query. I'm not sure what the normal routine of going about this is but just wanted to call it out.

Reply:
this is already handled. these tags are remapped at intake, so we are updating them to the new name that it is already being remapped to.

Reply:
(but yes, that is always something we should ask)

Reply:
What's the rationale for assigning the port number to a field called `host`? On its face I'm confused by that decision. I'm sensing there's a bunch of context that I lack.
```shell
$ git checkout -b add-network-destination-tags origin/add-network-destination-tags                       
branch 'add-network-destination-tags' set up to track 'origin/add-network-destination-tags' by rebasing.
Switched to a new branch 'add-network-destination-tags'
$ ack 'network\.destination\.name'                  
$ ack 'network\.destination\.ip'  
$ ack 'network\.destination\.port'
$
```